feeder/util/mqtt/authentication.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+import re
+
+from hbmqtt.plugins.authentication import BaseAuthPlugin
+from feeder.database.models import KronosGateways
+from secrets import token_hex
+
+
+local_username = "local_%s" % token_hex(8)
+local_password = token_hex(16)
+
+
+class PetnetAuthPlugin(BaseAuthPlugin):
+    username_regex = re.compile(r'^/pegasus:(?P<gateway_id>.*)$')
+
+    def __init__(self, context):
```

Comment:
You shouldn’t need this constructor. I know they have it in the library code, but a child class will just inherit its parent constructor, so there is no need to make our own if all we are doing is calling super.